interfaces/Matlab_Toolbox_Revamp/Class/Mixture.m

```diff
@@ -0,0 +1,328 @@
+classdef Mixture < handle
+    
+    properties
+        mixindex
+        phases
+        T
+        P
+    end
+    
+    methods
+        %% Mixture class constructor
+
+        function m = Mixture(phases)
+            % To construct a mixture, supply a cell array of phases and mole
+            % numbers:
+            %    >> gas = Solution('gas.cti');
+            %    >> graphite = Solution('graphite.cti');
```

Comment:
All examples should use `.yaml` extensions for the input files. I think it'd also be good if this used input files that actually exist (`gas.yaml` doesn't exist, and I don't recall if `graphite.yaml` does or not).